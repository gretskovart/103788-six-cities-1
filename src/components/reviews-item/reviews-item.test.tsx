import * as React from 'react';
import RewiewsItem from './reviews-item';
import renderer from 'react-test-renderer';

const mock = {
  id: `id`,
  date: `date`,
  dateString: `dateString`,
  comment: `comment`,
  rating: null,
  user: {
    avatarUrl: `avatarUrl`,
    name: `name`,
    id: `id`,
    isPro: true
  }
};

it(`<RewiewsItem /> renders correctly`, () => {
  const tree = renderer
    .create(
        <RewiewsItem
          review={mock}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
