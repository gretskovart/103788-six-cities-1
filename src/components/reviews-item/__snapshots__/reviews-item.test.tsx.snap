// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RewiewsItem /> renders correctly 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="avatarUrl"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        name
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "0%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        comment
      </p>
      <time
        className="reviews__time"
        dateTime="date"
      >
        dateString
      </time>
    </div>
  </li>
</ul>
`;
